cli/restore.go

```diff
@@ -107,10 +110,32 @@ func restore(cn *pbm.PBM, bcpName string, outf outFormat) error {
 	return waitForRestoreStatus(ctx, cn, name)
 }
 
-func pitrestore(cn *pbm.PBM, t, base string, outf outFormat) error {
+func parseTS(t string) (ts primitive.Timestamp, err error) {
+	if si := strings.Index(t, ","); si != -1 {
```

Comment:
can we use [`strings.SplitN()`](https://pkg.go.dev/strings#SplitN)? the code makes understanding hard and not much optimization in runtime